amida-tech/greyscale
# Feature/inba 887 888 jf moment memcache

#### What does this PR do?
Updates moment.js to remove vulnerability, upgrades the Node version and removes memcache.

#### Related JIRA tickets:
INBA-887, INBA-888.

#### How should this be manually tested?
Bump your Node version up to or between 6.10.0 to 6.12.2. Run `yarn cache clean` and `yarn install`... just make sure that memcache is not in node_modules anymore. 

Run tests, regression test everything and be sure to do `node seed.js`. 

@ElijahGeorge please check that the kubernetes documentation is correct. 

#### Background/Context

#### Screenshots (if appropriate):
